### PR TITLE
v1.4.16

### PR DESCRIPTION
**Pull request by jijojosephk**

### CODE REVIEW

File: com.github.IsmaelMartinez.teams_for_linux.yml [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-runtime-version: '22.08'
+runtime-version: '23.08'
 sdk: org.freedesktop.Sdk
 command: teams-for-linux
 separate-locales: false
 finish-args:
   - --share=ipc
   - --socket=wayland
   - --socket=x11
+  - --socket=fallback-x11
```
Comment (Michsior14) [inline com.github.IsmaelMartinez.teams_for_linux.yml:13]: This should replace `--socket=x11` as in #90 :)

**Comment by losuler** [comment]
Thought I would try this to see if it removed the new banner message https://github.com/flathub/com.github.IsmaelMartinez.teams_for_linux/issues/95 (which unfortunately it doesn't), but I thought I'd mention this doesn't appear to be working with the wayland socked enabled (just crashes before opening, as shown below).

```
$ flatpak run --branch=test com.github.IsmaelMartinez.teams_for_linux
[2:0404/142208.956267:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[WARN] No config file found, using default values
LaunchProcess: failed to execvp:
xdg-settings
[2:0404/142209.124323:ERROR:ozone_platform_x11.cc(244)] Missing X server or $DISPLAY
[2:0404/142209.124332:ERROR:env.cc(257)] The platform failed to initialize.  Exiting.
```

I'm only able to get it to launch if I disable the wayland socket (same errors as stable):

```
$ flatpak run --branch=test --nosocket=wayland com.github.IsmaelMartinez.teams_for_linux
[2:0404/143018.286914:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[WARN] No config file found, using default values
LaunchProcess: failed to execvp:
xdg-settings
[2:0404/143018.524788:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[ERROR] net::ERR_CONNECTION_REFUSED
[2:0404/143019.009182:ERROR:object_proxy.cc(576)] Failed to call method: org.freedesktop.ScreenSaver.GetActive: object_path= /org/freedesktop/ScreenSaver: org.freedesktop.DBus.Error.NotSupported: This method is not part of the idle inhibition specification: https://specifications.freedesktop.org/idle-inhibit-spec/latest/
```

File: com.github.IsmaelMartinez.teams_for_linux.yml (hunk 1 above)
Comment (jijojosephk) [reply com.github.IsmaelMartinez.teams_for_linux.yml:12]: I think this one is for runtime `23.08`. Replaced it now. Let's see. Thanks!

**Comment by jijojosephk** [comment]
App is not launching. Not sure it is `22.08` to `23.08` jump or `fallback-11`

**Comment by losuler** [comment]
Doesn't appear to be related to the base/runtime version. This latest build still doesn't launch for me with either:

```
flatpak run --branch=test com.github.IsmaelMartinez.teams_for_linux
flatpak run --branch=test --nosocket=fallback-x11 com.github.IsmaelMartinez.teams_for_linux
```

But it will launch with:

```
flatpak run --branch=test --nosocket=wayland com.github.IsmaelMartinez.teams_for_linux
```

**Comment by jijojosephk** [comment]
To me `fallback-11` is causing the problem. I'll try to upgrade to `23.08`

**Comment by losuler** [comment]
Oh I see what the issue is. You're missing the `--env=ELECTRON_OZONE_PLATFORM_HINT=auto` environment variable that was originally added in https://github.com/flathub/com.github.IsmaelMartinez.teams_for_linux/pull/90, but that was never added to this pull request.

I installed this previous test build https://github.com/flathub/com.github.IsmaelMartinez.teams_for_linux/pull/94#issuecomment-2036210606 and ran it with this command, and it launched:

```
flatpak run --branch=test --env=ELECTRON_OZONE_PLATFORM_HINT=auto com.github.IsmaelMartinez.teams_for_linux
```

**Comment by jijojosephk** [comment]
I don't know how `--env=ELECTRON_OZONE_PLATFORM_HINT=auto` will behave for everyone. Can you explain what is that flag?

**Comment by losuler** [comment]
It's explained here https://www.electronjs.org/docs/latest/api/environment-variables#electron_ozone_platform_hint-linux (as originally mentioned by @Mikaela in https://github.com/flathub/com.github.IsmaelMartinez.teams_for_linux/issues/72#issuecomment-2005814052).

Looks like it crashes because the Flatpak is trying to use the Wayland socket while Electron is trying to use x11, because `ELECTRON_OZONE_PLATFORM_HINT` is defaulting to the `x11` option. Not sure why their default wouldn't be `auto`.

**Pull request merged**